src/pal/tools/gen-buildsys-gcc.sh

```diff
@@ -80,55 +87,39 @@ for i in "$@"; do
     esac
 done
 
-OS=`uname`
+OS=$(uname)
 
 locate_gcc_exec() {
+  ENV_KNOB="CLR_$(echo "$1" | tr '[:lower:]' '[:upper:]')"
```

Comment:
@am11 any particular reason you need the `ENV_KNOB` mechanism here?

Reply:
When the consumer with custom toolchain wants to override particular component(s), `CLR_CC`, `CLR_CXX`, `CLR_NM`, `CLR_LINK`, `CLR_AR`, `CLR_RANLIB`, `CLR_OBJDUMP`, `CLR_OBJCOPY` knobs can be used without modifying the buildsync script.